File: beecrowd/1005.cpp
// created by mostafizzfuad
#include <bits/stdc++.h>
using namespace std;
int main() {

    double a, b;
    cin >> a >> b;

    double sumOfWeights = 3.5 + 7.5;
    double avg = ((a * 3.5) + (b * 7.5)) / sumOfWeights;

    printf("MEDIA = %.5lf\n", avg);

    return 0;
}

File: beecrowd/1005.cs
// created by mostafizzfuad
using System;
class URI
{
    static void Main(string[] args)
    {

        double a = Convert.ToDouble(Console.ReadLine());
        double b = Convert.ToDouble(Console.ReadLine());

        double sumOfWeight = 3.5 + 7.5;
        double avg = ((a * 3.5) + (b * 7.5)) / sumOfWeight;

        Console.WriteLine("MEDIA = " + avg.ToString("0.00000"));

    }
}